### PR TITLE
11/08/2015 09:19am

### PR DESCRIPTION
**Pull request by Flotitan**

### CODE REVIEW

**Comment by laurencee** [comment]
You should probably rebase and then squash these 32 commits into a single commit since most of the commits appear to be 1 line changes.

**Comment by Flotitan** [comment]
Sorry, but what exactly do you mean and how do I do it? I'm working with Github for the first time, so I'm still learning.

**Comment by ghost** [comment]
The repository you're editing off of has been changed at some point since you've created this pull request. Because of this, IMC merging this branch might result in some conflicts. This is why you have to "rebase" it by closing this pull request and submitting a new one based off of the new repository.

Also, when submitting pull requests, it's discouraged (though not strictly enforced) to make only a single line edit at a time, and you have to make separate pull requests for each file you've edited. You've edited files QUEST_LV_100.tsv, QUEST.tsv, and ITEM.tsv, so when you're resubmitting this, make separate pull requests for each of those files.

Please re-read the guidelines (https://github.com/Treeofsavior/EnglishTranslation/wiki/Guidelines) and the wiki (https://github.com/Treeofsavior/EnglishTranslation/wiki) before making any more edits. Thank you.

**Comment by laurencee** [comment]
Look at the following for rebasing your changes over the latest changes from the IMC repository:
http://stackoverflow.com/a/7244456

Take a look at the following for squashing commits into a single commit: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

You may find this easier to do in a tool like SourceTree if you're not familiar with the git command lines do.
You can practice on a repository first by just literally copying and pasting the folder on your PC, and running the git commands on that copied folder. That way if something goes wrong you can delete the copy, make a new one and try again.

**Comment by laurencee** [comment]
I don't believe you even need to make a new pull request when you do this, as long as you force push your updated rebased & squashed commits to your fork, this pull request should automatically reflect the changes.
**EDIT: As @Senhart mentioned, you should only have 1 file per pull request so you will have to make new branches with the files updated in the other branches and then submit pull requests on those branches.**
1. First you'll want to rebase your current changes over the latest IMC repository changes
2. Squash all your commits into 1 commit. 
3. Make a copy of and then revert the changes to ITEM.tsv, QUEST.tsv, ETC.tsv and QUEST_JOBSTEP.tsv so you are only left with changes to QUEST_LV_0100.tsv
4. For each of the above reverted files, create a new branch and commit the 1 of those files
5. Submit pull requests for each of those new branches

Give this a go first though and see how you go, there are plenty of articles online about using git.

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -6134,11 +6134,11 @@ QUEST_LV_0100_20150317_006133	Give crutches to Kayhil
 QUEST_LV_0100_20150317_006134	Moze was actually making crutches for Kayhil. Give Moze's crutches to Kayhil.
 QUEST_LV_0100_20150317_006135	Talk to Follower Domas
 QUEST_LV_0100_20150317_006136	Goddess Saule's Follower Domas needs help in Gate Route.
-QUEST_LV_0100_20150317_006137	Kill Thornball and get its debris
+QUEST_LV_0100_20150317_006137	Kill Dunball and get its debris
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:6137]: Kill -> Defeat

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -6134,11 +6134,11 @@ QUEST_LV_0100_20150317_006133	Give crutches to Kayhil
...
 QUEST_LV_0100_20150317_006138	
 QUEST_LV_0100_20150317_006139	Give it to Domas
 QUEST_LV_0100_20150317_006140	Got the debris of Thornball to use for burning the forest. But won't the whole forest burn like this? Anyway, give it to Domas.
-QUEST_LV_0100_20150317_006141	Kill Thornball and get %s
+QUEST_LV_0100_20150317_006141	Kill Dunball and get %s
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:6141]: Kill -> Defeat

**Pull request closed without merging**